### PR TITLE
Fixtransition

### PR DESCRIPTION
**Pull request by winsvega**
tests
https://github.com/ethereum/tests/pull/588

### CODE REVIEW

File: libp2p/Common.h [hunk 1]
```diff
@@ -231,7 +231,7 @@ class Node
 
     NodeID const& address() const { return id; }
 
-    explicit operator bool() const { return bool{id}; }
+    explicit operator bool() const { return (bool)id; }
```
Comment (chfast) [inline libp2p/Common.h]:
```suggestion
    explicit operator bool() const { return static_cast<bool>(id); }
```
This must go to a separate commit.

File: libethashseal/genesis/test/ByzantiumToConstantinopleFixAt5Test.cpp [hunk 2]
```diff
@@ -26,7 +26,7 @@ static std::string const c_genesisInfoByzantiumToConstantinopleFixAt5Test = std:
 		"EIP150ForkBlock": "0x00",
 		"EIP158ForkBlock": "0x00",
 		"byzantiumForkBlock": "0x00",
-		"constantinopleForkBlock": "0x00",
+		"constantinopleForkBlock": "0x05",
```
Comment (chfast) [inline libethashseal/genesis/test/ByzantiumToConstantinopleFixAt5Test.cpp]: Please explain in the commit message what is changed and why.

Comment (winsvega) [reply libethashseal/genesis/test/ByzantiumToConstantinopleFixAt5Test.cpp]: because Constantinople and ConstantinopleFix activate at the same time on mainnet

Comment (chfast) [reply libethashseal/genesis/test/ByzantiumToConstantinopleFixAt5Test.cpp]: > in the commit message

**Comment by chfast** [comment]
Can be merged, thanks.
